### PR TITLE
Document libreswan CVEs in additional packages (#33)

### PR DESCRIPTION
**Pull request by bleader**
Add detail about the CVEs possiblly impacting our version of libreswan,
explaining what these could lead to and criticality of the issue.

Signed-off-by: David Morel <david.morel@vates.fr>

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [x] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [x] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/additionalpackages.md [hunk 1]
```diff
@@ -49,7 +49,30 @@ The controller domain is not an all-purpose Linux system. It must remain minimal
...
+- [CVE-2023-38712](https://libreswan.org/security/CVE-2023-38712/CVE-2023-38712.txt): Invalid IKEv1 repeat IKE SA delete causes crash and libreswan to restart
+- [CVE-2023-38710](https://libreswan.org/security/CVE-2023-38710/CVE-2023-38710.txt): Invalid IKEv2 REKEY proposal causes libreswan to restart
+
+Patches for these are not really backportable, we therefore kept it as is and target updating packages in the next major release.
+
+You can find information about all Libreswan CVEs on their [security page](https://libreswan.org/security/).
+
+##### Resulting security issue
+
+The `pluto` daemon may be crashed by malformed IKE (both v1 and v2) delete/notify requests, resulting in a DDoS on the keying service. If the daemon is restarted in loop quickly enough, this could as well lead to a DDoS of the whole host.
```
Comment (stormi) [inline docs/additionalpackages.md:66]: Is it DDoS (Distributed Denial of Service), or just DoS?

Comment (bleader) [reply docs/additionalpackages.md:66]: I think it is indeed DoS.

Comment (bleader) [reply docs/additionalpackages.md]: I changed it.

**Review by stormi — APPROVED** [review]

**Comment by stormi** [comment]
PR manually integrated to our new docs. See commit reference above.

**Pull request closed without merging**